Bottom: 4769f404cfecff60c1ea9a765d6a0d1024ebb081
Top:    f2f75c163c053077c440cacf9cace3dd14913e6f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-05 22:49:13 +0000

Refresh of patch-out-non-full-screen

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index afe7d1f..f62f522 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -2349,11 +2349,12 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
     object3d_test_init ();
   }
 
-  printf ("Expose event at (%i,%i): %i x %i\n", ev->area.x, ev->area.y, ev->area.width, ev->area.height);
+//  printf ("Expose event at (%i,%i): %i x %i\n", ev->area.x, ev->area.y, ev->area.width, ev->area.height);
 
   /* For some reason we get horrible glitching when we don't repaint the whole event region...
    * It also appears that we get spurious small repaints interleaved with full area re-paints,
-   * so lets try just droping the small ones
+   * so lets try just droping the small ones.
+   * NB: Currently, all our programmatically triggered repaints cover the whole region.
    */
   if (ev->area.x != 0 || ev->area.y != 0)
     return; /* XXX? */
